contracts/validator-manager/ValidatorMessages.sol

```diff
@@ -230,65 +267,169 @@ library ValidatorMessages {
         pure
         returns (ValidationPeriod memory)
     {
-        if (input.length != 134) {
-            revert InvalidMessageLength();
-        }
+        uint32 index = 0;
+        ValidationPeriod memory validation;
 
         // Unpack the codec ID
-        uint16 codecID;
-        for (uint256 i; i < 2; ++i) {
-            codecID |= uint16(uint8(input[i])) << uint16((8 * (1 - i)));
-        }
-        if (codecID != CODEC_ID) {
-            revert InvalidCodecID();
+        {
```

Comment:
Local variables are put into inner scopes to avoid max stack depth errors.

Reply:
can we add a comment for this?